WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -715,40 +715,6 @@ d_sunni_cuman_company = {
 	mercenary_type = muslim_cuman_company_composition
 }
 
-d_sunni_berber_company = { 
-	color = { 85 155 33 }
-	color2 = { 255 255 255 }
-
-	capital = 833 # Atlas
-
-	# Hire Trigger
-	allow = {
-		always = yes
-	}
-	
-	# Parent Religion 
-	religion = valentinianism
-	culture = african_romance
-	
-	mercenary = yes
-
-	title = "CAPTAIN"
-	foa = "CAPTAIN_FOA"
-
-	# Always exists
-	landless = yes
-	
-	# Cannot be held as a secondary title
-	primary = yes
-	
-	# Cannot be vassalized
-	independent = yes
-	
-	strength_growth_per_century = 1.00
-	
-	mercenary_type = berber_company_composition
-}
-
```

Comment:
I moved the "Knights of the VALENTINIANS" over to the Holy Orders because it's a Valentinian Holy Order.